get_subscription.py

```diff
@@ -87,12 +87,14 @@ def mount_get_root(mount):
 args = parser.parse_args()
 
 try:
-    appids = [app_to_sub[s] for s in args.appids]
+    subids = [app_to_sub[a] for a in args.appids if re.match('^\w+$', a)]
```

Comment:
let's split it when we need to use it. also this should be fine, ids are alphanum